priv/perf/config/dev.exs

```diff
@@ -1,6 +1,28 @@
 use Mix.Config
 
+config :ethereumex,
+  url: System.get_env("ETHEREUM_RPC_URL")
+
 config :load_test,
-  child_chain_url: "https://dev-7c3f796-childchain-ropsten-01.omg.network/",
-  watcher_security_url: "https://dev-7c3f796-watcher-ropsten-01.omg.network/",
-  watcher_info_url: "https://dev-7c3f796-watcher-info-ropsten-01.omg.network/"
+  child_chain_url: System.get_env("CHILD_CHAIN_URL") || "https://stress-e043a92-childchain-ropsten-01.omg.network",
```

Comment:
hmm, wait why is dev setting to stress?
Also why the removal of the other endpoints. Btw, the dev endpoints have changed due to new release 😢 shall update to the latest one.

Reply:
I noticed that to. Our old stress test env is down. I am not sure which one should I use in settings now.

Reply:
This one I think: https://github.com/omisego/devops/wiki/DevOps-Wiki#dev-a69c763-ropsten-01

Reply:
@boolafish  I removed other endpoints because they are not needed now. This PR moves everything *but* the prev. test scenarios. It's basically what's needed to run utxo load test. The missing parts are just a couple of settings, but I don't like to commit something that's not needed now.

Reply:
dev endpoint is used in the CI on master branch with smoke test now. The intention is to push load after CD.

Reply:
@boolafish I removed hardcoded addresses pointing to stress and dev envs. They are read from system vars now. I think that's cleaner than having them hardcoded as they might change independently to the code. What you think Andy?

Reply:
If we are running test on CI/CD against real env, you will need to change either code config or CI config anyway whenever there is a contract sha changes. I personally don't think config for different env is considered unclean and it is slightly more convenient for dev to run against real env locally. 

But I don't have strong opinion.

Reply:
I would like that both of you ask devops for getting fixed environment names but also stress (no pun intended) the importance of a fixed environment domain naming and what problems it's causing us. cc @achiurizo

Reply:
hi @jbunce 👆
But yeah, I think a fixed endpoint is necessary if we want to move forward CI/CD pipeline with automated tests hitting the env instead of docker-compose env. I think it is fine the underneath env is actually moving (chaining sha) under the endpoint. But fixed endpoint is quite useful.